topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorMockTest.java

```diff
@@ -67,22 +65,14 @@
 
     // TODO this is all in common with TOIT, so factor out a common base class
 
-    @BeforeAll
-    public static void before() {
+    @BeforeEach
```

Comment:
To reviewers: Github diff might seem like a mess. Basically it is just moving `vertx` instantiation from `BrforeAl`l to `BeforeEach` (similarly with closing it).